crates/matrix-sdk-crypto/src/olm/group_sessions/sender_data.rs

```diff
@@ -74,6 +74,14 @@ pub enum SenderData {
         /// If false, we had simply accepted the key as this user's latest
         /// key.
         master_key_verified: bool,
+
+        #[serde(default)]
+        /// Whether the user's master key was previously verified, but the
+        /// current master key is not.
```

Comment:
if this is the actual behaviour (and I'm not sure that it is), it needs a name that reflects it.